lib/components/FileAttachment/index.jsx

```diff
@@ -0,0 +1,181 @@
+import React, { useEffect, useRef, useState } from "react";
+
+import "@uppy/core/dist/style.css";
+import "@uppy/progress-bar/dist/style.css";
```

Comment:
Even though these are part of uppy.io docs, we don't need these imports here since we have a custom upload UI.